platforms/common/cpp/core/JitProgram.cpp

```diff
@@ -376,7 +374,7 @@ bool FlowJitProgram::compile()
 
     // Initialize generation
     next_code_off = 0;
-    next_data_off = committed_data_off = MAX_CODE_MEMORY;
+	next_data_off = committed_data_off = std::max(memory_limit, (unsigned long)flow_code.size() * 25);
```

Comment:
Keep a 256mb minimum size, just to ensure we do not break existing code in some random circumstance. Then we are sure this is a monotoneous improvement.

Reply:
See no point in this, since we had problems only with rhapsode, but ok. Done.